lib/Conversion/TritonGPUToLLVM/MemoryOpToLLVM.cpp

```diff
@@ -126,7 +126,7 @@ struct LocalLoadOpConversion : public ConvertOpToLLVMPattern<LocalLoadOp> {
             LinearEncodingAttr>(dstLayout))
       return true;
     if (auto dot = dyn_cast<DotOperandEncodingAttr>(dstLayout)) {
-      if (isa<MmaEncodingTrait>(dot.getParent()))
+      if (isa<MmaEncodingTrait, BlockedEncodingAttr>(dot.getParent()))
         return true;
     }
     return false;
```

Comment:
Now we can remove the FIXME and this function altogether, as all the layouts are supported :)

Reply:
I like removing code :slightly_smiling_face: 
done!